mkdocs.yml

```diff
@@ -29,69 +29,64 @@ markdown_extensions:
-        - Media Azure: OrchardCore.Modules/OrchardCore.Media.Azure/README.md
-        - Menu: OrchardCore.Modules/OrchardCore.Menu/README.md
-        - Navigation: OrchardCore.Modules/OrchardCore.Navigation/README.md
-        - Indexing: OrchardCore.Modules/OrchardCore.Indexing/README.md
-        - Lucene: OrchardCore.Modules/OrchardCore.Lucene/README.md
-        - Queries: OrchardCore.Modules/OrchardCore.Queries/README.md
-        - Taxonomies: OrchardCore.Modules/OrchardCore.Taxonomies/README.md
-        - Templates: OrchardCore.Modules/OrchardCore.Templates/README.md
-        - Themes: OrchardCore.Modules/OrchardCore.Themes/README.md
-        - Liquid: OrchardCore.Modules/OrchardCore.Liquid/README.md
-        - Custom Settings: OrchardCore.Modules/OrchardCore.CustomSettings/README.md
-        - Widgets: OrchardCore.Modules/OrchardCore.Widgets/README.md
-        - Flows: OrchardCore.Modules/OrchardCore.Flows/README.md
-        - Forms: OrchardCore.Modules/OrchardCore.Forms/README.md
-        - Admin Menu: OrchardCore.Modules/OrchardCore.AdminMenu/README.md
-    - Core Modules:
-        - Configuration: OrchardCore/OrchardCore/Environment/Shell/README.md
-        - Data: OrchardCore/OrchardCore.Data/README.md
-        - Dynamic Cache: OrchardCore.Modules/OrchardCore.DynamicCache/README.md
-        - GraphQL: OrchardCore.Modules/OrchardCore.Apis.GraphQL/README.md
-        - GraphQL queries: OrchardCore/OrchardCore.Apis.GraphQL.Abstractions/README.md
-        - Health Check: OrchardCore.Modules/OrchardCore.HealthChecks/README.md
-        - Localization: OrchardCore.Modules/OrchardCore.Localization/README.md
-        - Logging Serilog: OrchardCore/OrchardCore.Logging.Serilog/readme.md
-        - OpenId: OrchardCore.Modules/OrchardCore.OpenId/README.md
-        - Placement: OrchardCore/OrchardCore.DisplayManagement/README.md
-        - Recipes: OrchardCore.Modules/OrchardCore.Recipes/README.md
-        - Resources: OrchardCore.Modules/OrchardCore.Resources/README.md
-        - Roles: OrchardCore.Modules/OrchardCore.Roles/README.md
-        - Scripting: OrchardCore.Modules/OrchardCore.Scripting/README.md
-        - Setup: OrchardCore.Modules/OrchardCore.Setup/README.md
-        - Tenants: OrchardCore.Modules/OrchardCore.Tenants/README.md
-        - Workflows: OrchardCore.Modules/OrchardCore.Workflows/README.md
-        - DataProtection (Azure Storage): OrchardCore.Modules/OrchardCore.DataProtection.Azure/README.md
-    - Authentication Modules:
-        - Microsoft: OrchardCore.Modules/OrchardCore.Microsoft.Authentication/README.md
-        - Facebook: OrchardCore.Modules/OrchardCore.Facebook/README.md
-        - Twitter: OrchardCore.Modules/OrchardCore.Twitter/README.md
-        - GitHub: OrchardCore.Modules/OrchardCore.GitHub/README.md
-        - Google: OrchardCore.Modules/OrchardCore.Google/README.md
+        - Create a modular application: docs/guides/create-modular-application-mvc/index.md
```

Comment:
A reason to remove 'ing' ?

Reply:
To have it on one line in the left menu on RTD